ext/ext/count_min_sketch.cpp

```diff
@@ -143,4 +145,53 @@ unsigned int CountMinSketch::hashstr(const char *str) {
   return hash;
 }
 
+void CountMinSketch::swap(CountMinSketch& rhs)
+{
+  std::swap(eps, rhs.eps);
+  std::swap(this->gamma, rhs.gamma);
+  std::swap(total, rhs.total);
+  std::swap(w, rhs.w);
+  std::swap(d, rhs.d);
+  std::swap(C, rhs.C);
+  std::swap(hashes, rhs.hashes);
+}
+
+// XXX - this function does not currently convert everything to network byte order
+void CountMinSketch::dump(std::ostream& os) const throw(std::runtime_error)
+{
+  int i=0;
+  os.write((char*)&eps, sizeof(eps));
```

Comment:
why not use https://www.boost.org/doc/libs/1_59_0/libs/endian/doc/buffers.html instead since you are already using boost?